src/openqaoa-braket/backends/gates_braket.py

```diff
@@ -0,0 +1,94 @@
+from typing import Callable
+
+from braket.circuits import gates, Circuit
+
+from openqaoa.qaoa_components.ansatz_constructor.rotationangle import RotationAngle
+import openqaoa.qaoa_components.ansatz_constructor.gates as gates_core
+
+class BraketGateApplicator(gates_core.GateApplicator):
+
+    BRAKET_OQ_GATE_MAPPER = {
+        gates_core.RZ.__name__: gates.Rz.rz,
+        gates_core.RX.__name__: gates.Rx.rx,
+        gates_core.RY.__name__: gates.Ry.ry,
+        gates_core.CX.__name__: gates.CNot.cnot,
+        gates_core.CZ.__name__: gates.CZ.cz,
+        gates_core.RXX.__name__: gates.XX.xx,
+        gates_core.RZZ.__name__: gates.ZZ.zz,
+        gates_core.RYY.__name__: gates.YY.yy,
+        gates_core.CPHASE.__name__: gates.CPhaseShift.cphaseshift,
+        gates_core.RiSWAP.__name__: gates.XY.xy
+    }
+
+    def __init__(self):
+        self.library = 'braket'
+
+    def gate_selector(self, gate: gates_core.Gate) -> Callable:
```

Comment:
The ENUM strategy (I think) will also remove the need for these checks (they are basically built-in within the ENUM structure)